src/app/directives/sidebar.js

```diff
@@ -484,13 +484,25 @@
           $scope.responseDetails = false;
           $scope.response        = {};
 
+          var mockingServiceDetector = /(?:(?:mocksvc\.[a-z\.]*)|(?:[a-z]+\.anypoint\.))mulesoft\.com\/.*mocks\/([0-9a-zA-Z-]+)(\/(.+))?/;
+          var mockingService2Detector = /(?:(?:[a-z.]*anypoint\.mulesoft\.com)|(?:localhost:3000))\/mocking\/api\/v1\/links\/([0-9a-zA-Z-]+)(\/(.*))?$/;
+
```

Comment:
The patterns are duplicated in two files, is there a way to have them in one place?